usr.sbin/mptutil/mpt_volume.c

```diff
@@ -246,9 +246,9 @@ volume_cache(int ac, char **av)
 	Settings = volume->VolumeSettings.Settings;
 
 	NewSettings = Settings;
-	if (strncmp(av[2], "enable", sizeof("enable")) == 0)
+	if (strncmp(av[2], "enable", sizeof("enable") - 1) == 0)
```

Comment:
Could this be changed to use `strlen()` instead?

Reply:
@AtariDreams : I agree with @delphij. Is it ok if we/I apply his suggestion and merge the change on your behalf?

Reply:
FWIW, it's functionally the same as written, unless `sizeof(av[2])` becomes a `char*` quantity (in which case it will always chop the string off at 8 characters on 64-bit platforms :D...).

Reply:
Yes that is okay! @ngie-eign